plans/multi-craft-labor-request-enhancement.md

```
@@ -0,0 +1,1742 @@
+**6.6 Add Status Tracking**
+- Extend `labor_request_notifications` table with status field:
+  ```sql
+  ALTER TABLE labor_request_notifications
+  ADD COLUMN viewed_at TIMESTAMPTZ,
+  ADD COLUMN status TEXT DEFAULT 'new'
+    CHECK (status IN ('new', 'viewed', 'responded', 'archived'));
+  ```
+- Update status when agency opens request detail
+- Create API endpoint: `PATCH /api/labor-requests/notifications/[id]` to update status
+
+**6.7 Integrate with Messaging System (Optional)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)